README.md

```diff
@@ -33,22 +36,44 @@ must succeed (sometimes more than once) before the warning symbol disappears.
 
 ## Home Assistant Control
 
-Copy the carrier_infinity/ directory under homeassistant/ here into the
-config/custom_components/ directory of your Home Assistant installation. Add
-configuration such as the following to tell Home Assistant where the HTTP server
+Copy the custom_components/carrier_infinity/ directory under homeassistant/ 
+here into the config/custom_components/ directory of your Home Assistant 
+installation. Alternatively, download via HACS.
+
+Add configuration such as the following to tell Home Assistant where the HTTP server
 is running:
 
     climate:
       - platform: carrier_infinity
-        host: infinity.internal.cus
+        host: HomeAssistantIP
+        port: 5000
+    
+    climate:
+      - platform: carrier_infinity
+        host: my.spr.scrt.ip
         port: 5000
+        pushover_user: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        pushover_token: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        zone_names:
+          - House_Furnace_Carr
+        scan_interval: 300
 
 Restart Home Assistant and the thermostat will appear.  Initially Home Assistant
 may not show real data from the actual thermostat.  The thermostat must check in
 with the HTTP server before any status information can be determined.  Likewise,
 temperature and other settings cannot be adjusted until the thermostat has sent
 its current configuration to the HTTP server.
 
+On restart of HomeAssistant the integration hangs and will await the thermostat 
+to post before completing the startup.
+
+# Pushover
```

Comment:
I think that pushover support should be done by scripts inside Homeassistant instead of in this integration.  Initially when I had the code separated into two pieces with the proxy being a completely separate server it would make sense to put pushover directly in the proxy.  However, now that all of the code is tightly coupled to Homeassistant anyway, rather than duplicating that support here it would be better to make use of the existing integration for that aspect.

Reply:
As it's already in here, would rather just leave it.

Reply:
Implemented it anyway as I was curious how much extra work I've been doing.